docker/regtest/docker-compose.yml

```diff
@@ -124,10 +124,12 @@ services:
       - 8080  # payjoin server
       - 62601 # obwatch
       - 27183 # joinmarketd
-      - 28183 # jmwalletd
+      - 28183 # jmwalletd api
+      - 28283 # jmwalletd websocket
```

Comment:
This doesn't look right :)

Reply:
Hey @dennisreimann .. Thanks for looking over it (still a draft!)

Can you elaborate?

From the docs:
> [...] which with defaults will start serving the RPC over https:// on port 28183, and a (secure) websocket server (wss://) on port 28283.

https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/v0.9.5/docs/JSON-RPC-API-using-jmwalletd.md#introduction---how-to-start-the-server

Reply:
> Yikes, it was just me being blind. Sorry for bothering!

You do not bother me. Quite the opposite. It's good to know you are watching over it.
I'd be happy if you were a reviewer : D
Will ping you once it is ready!